src/main/java/uk/gov/hmcts/reform/ccd/documentam/controller/endpoints/CaseDocumentAmController.java

```diff
@@ -331,15 +312,24 @@ public UploadResponse uploadDocuments(
             + ".extractIds(#caseDocumentsMetadata.documentHashTokens)",
         caseId = "#caseDocumentsMetadata.caseId"
     )
-    public ResponseEntity<Object> patchMetaDataOnDocuments(
+    public ResponseEntity<PatchDocumentMetaDataResponse> patchMetaDataOnDocuments(
         @ApiParam(value = "", required = true)
-        @Valid @RequestBody CaseDocumentsMetadata caseDocumentsMetadata,
+        @Valid @RequestBody final CaseDocumentsMetadata caseDocumentsMetadata,
+
+        final BindingResult bindingResult,
+
         @ApiParam(value = "S2S JWT token for an approved micro-service", required = true)
-        @RequestHeader(SERVICE_AUTHORIZATION) String s2sToken
-    ) {
-        validationUtils.validate(caseDocumentsMetadata.getCaseId());
+        @RequestHeader(SERVICE_AUTHORIZATION) final String s2sToken) {
+
+        handleErrors(bindingResult);
+
         documentManagementService.validateHashTokens(caseDocumentsMetadata.getDocumentHashTokens());
 
+        //validate the service authorization for first document in payload
```

Comment:
Probably merge issue. Please remove this as `documentMetadata` is no longer used in this method.